platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheGradlePropertiesIntegrationTest.groovy

```diff
@@ -330,4 +331,212 @@ class ConfigurationCacheGradlePropertiesIntegrationTest extends AbstractConfigur
         cleanup:
         System.clearProperty(systemProp)
     }
+
+    def "reuses cache when unused project property changes on command-line"() {
+        buildFile """
+            tasks.register("some")
+        """
+
+        when:
+        configurationCacheRun "some", "-Pfoo=one"
```

Comment:
💅 Add a new line in between `when` and `then` for consistency with rest of the tests in the file

Reply:
Thanks, done.